toolkit/tools/internal/csvparser/csvparser.go

```diff
@@ -0,0 +1,78 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+// Parses the timestamps stored in csv files and print them
+// to the terminal at the end of the build.
+
+package csvparser
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+	"strings"
+	"time"
+)
+
+var (
+	timeArray [][]string // A 2-D array used to store all timestamps from csvs.
+	// A list of csv files to be parsed.
+	files = []string{"/imageconfigvalidator.csv", "/imagepkgfetcher.csv", "/imager.csv", "/roast.csv"}
```

Comment:
I don't like the idea of keeping a hard-coded list of files here. It's creating an unhealthy (hard to discover and fixed) relationship between the parser and multiple other tools. It should be up to the tools making use of the timestamps to decide when/if to use the parser.